modules/media/jsx/uploadForm.js

```diff
@@ -340,9 +340,23 @@ var MediaUploadForm = React.createClass({
    */
   setFormData: function(formElement, value) {
     // Only display visits and sites available for the current pscid
+    var visitLabel=this.state.formData.visitLabel;
     if (formElement === "pscid" && value !== "") {
       this.state.Data.visits = this.state.Data.sessionData[value].visits;
       this.state.Data.sites = this.state.Data.sessionData[value].sites;
+      if (visitLabel !== null) {
+        this.state.Data.instruments =
+            this.state.Data.sessionData[value].instruments[visitLabel];
+      } else {
+        this.state.Data.instruments =
+            this.state.Data.sessionData[value].instruments['all'];
+      }
+    }
+    var pscid = this.state.formData.pscid;
+    if (formElement === "visitLabel" &&
+        value !== "" && pscid !== null) {
```

Comment:
I think the if condition is less than 80 chars so it doesn't have to go on a new line